swagger.yml

```diff
@@ -2239,34 +2239,38 @@ paths:
                     description: full paged
                     properties:
                       content:
-                        type: array
-                        items:
-                          $ref: '#/components/schemas/Database'
+                        type: object
```

Comment:
Is this true?
It's still returned an array with the object database inside, not an object!